### DISCUSSION TITLE
is there a tutorial for getting this setup?

### DISCUSSION THREAD

**Discussion by SugoiShades** [MrBrax/LiveStreamDVR#500] [Q&A]
I have a feeling I could stumble my way through setting most of this up, but nothing has tooltips or anything so I have no idea where to start? I think I can setup a NGINX rev proxy on my opnsense box but I have no idea how that would change how I connect to twitch as I'll use the same public IP as my server would connect to twitch with and I don't pay anyone for SSL certs so I don't think I can set anything like that up, does that mean I just can't use this? I'm so lost

**Comment by DajWaj** [comment]
@SugoiShades Did you get it already? (i need help🙏) I am also struggling with setting this one up. I don't understand why when i found other app with the same features but without the chat feature it installed with 1 click and here i need to sit few hours just for the stupid chat. Im trying with help of chat gpt but I think i'll pass and give up.

**Reply by MrBrax** [reply]
docker is preferred, use something like caddy or nginx on your firewall/outward facing host, use letsencrypt for https
